### PR TITLE
Fix 404 not found

### PR DESCRIPTION
**Pull request by xichengliudui**
fix some 404 not found

### CODE REVIEW

File: content/en/docs/contribute/generate-ref-docs/federation-api.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ and your `$GOPATH` environment variable must be set.
 * You need to know how to create a pull request to a GitHub repository.
 Typically, this involves creating a fork of the repository. For more
 information, see
-[Creating a Documentation Pull Request](/docs/home/contribute/create-pull-request/).
+[Creating a Documentation Pull Request](https://github.com/kubernetes/website/blob/release-1.11/content/en/docs/home/contribute/create-pull-request.md).
```
[the remark this answers is not in the document]
Comment (xichengliudui) [reply content/en/docs/contribute/generate-ref-docs/federation-api.md]: Is that ok?

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/contribute/generate-ref-docs/federation-api.md]: @xichengliudui Yes, I think so. The original link is now pointing to nowhere and the key here is to point contributors to some general guidelines on contribution.

[the remark this answers is not in the document]
Comment (xichengliudui) [reply content/en/docs/contribute/generate-ref-docs/federation-api.md]: @tengqm However, these files do not exist under master branch, and I must point to release-1.11

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/contribute/generate-ref-docs/federation-api.md]: No. They are there: https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/start.md

File: content/en/docs/contribute/generate-ref-docs/kubernetes-components.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ reference documentation for tools and components in the
 * You need to know how to create a pull request to a GitHub repository.
 Typically, this involves creating a fork of the repository. For more
 information, see
-[Creating a Documentation Pull Request](/docs/home/contribute/create-pull-request/).
+[Creating a Documentation Pull Request](https://github.com/kubernetes/website/blob/release-1.11/content/en/docs/home/contribute/create-pull-request.md).
```
Comment (stewart-yu) [inline content/en/docs/contribute/generate-ref-docs/kubernetes-components.md]: The same

**Pull request closed without merging**